changelogs/fragments/5502-xfconf-facts-deprecation.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Can we change this to `removed_features` and change the description so it's clear what exactly is removed/changed?

Reply:
Sure, should we explicitly mention that commit when we disabled facts in that description?

Reply:
done